sds_data_manager/lambda_code/SDSCode/database/config.py

```diff
@@ -0,0 +1,21 @@
+"""This stores database configuration such as database URI"""
+import json
+import os
+
+import boto3
+
+
+def get_db_uri():
+    """Create DB URI from secret manager.
+    Returns:
+        str: DB URI
+    """
+    secret_name = os.environ["SECRET_NAME"]
+    session = boto3.session.Session()
+    client = session.client(service_name="secretsmanager")
+    secret_string = client.get_secret_value(SecretId=secret_name)["SecretString"]
+    secret = json.loads(secret_string)
```

Comment:
```suggestion
    db_config = json.loads(secret_string)
```

I think this name is a bit more descriptive on what kind of data it is storing, but that is just personal preference